content/scalability/docs/control-plane.md

```diff
@@ -93,3 +93,86 @@ autoscalingGroups:
+
+
+### Monitoring APF and Dropped Requests:
+
+To confirm if any requests are being dropped due to APF, the API Server metrics for ```apiserver_flowcontrol_rejected_requests_total``` can be monitored to check the impacted FlowSchemas and PriorityLevelConfigurations. For example, this metric shows that 100 requests from the service-accounts FlowSchema were dropped due to requests timing out in workload-low queues:
+
+```
+% kubectl get --raw /metrics | grep rapiserver_flowcontrol_rejected_requests_total
+apiserver_flowcontrol_rejected_requests_total{flow_schema="service-accounts",priority_level="workload-low",reason="time-out"} 100
+```
+To check how close a given PriorityLevelConfiguration is to receiving 429s or experiencing increased latency due to queuing, you can compare the difference between the concurrency limit and the concurrency in use. In this example, we have a buffer of 100 requests.
+
+```
+% kubectl get --raw /metrics | grep 'apiserver_flowcontrol_request_concurrency_limit.*workload-low'
+apiserver_flowcontrol_request_concurrency_limit{priority_level="workload-low"} 245
+
+% kubectl get --raw /metrics | grep 'apiserver_flowcontrol_request_concurrency_in_use.*workload-low'
+apiserver_flowcontrol_request_concurrency_in_use{flow_schema="service-accounts",priority_level="workload-low"} 145
+```
+To check if a given PriorityLevelConfiguration is experiencing queuing but not necessarily dropped requests, the metric for ```apiserver_flowcontrol_current_inqueue_requests``` can be referenced:
+```
+% kubectl get --raw /metrics | grep 'apiserver_flowcontrol_current_inqueue_requests.*workload-low'
+apiserver_flowcontrol_current_inqueue_requests{flow_schema="service-accounts",priority_level="workload-low"} 10
+```
+
+Other useful Prometheus metrics include:
+- apiserver_flowcontrol_dispatched_requests_total
+- apiserver_flowcontrol_request_execution_seconds
+- apiserver_flowcontrol_request_wait_duration_seconds
+
+See the upstream documentation for a complete list of [APF metrics](https://kubernetes.io/docs/concepts/cluster-administration/flow-control/#observability).
+
+### Preventing Dropped Requests:
+
+When APF is dropping requests due to a given PriorityLevelConfiguration exceeding its maximum number of allowed inflight requests, clients in the affected FlowSchemas have to decrease the number of requests executing at a given time. This can be accomplished by reducing the total number of requests made over the period where 429s are occurring. Note that long-running requests such as expensive list calls are especially problematic because they count as an inflight request for the entire duration they are executing. Reducing the number of these expensive requests or optimizing the latency of these list calls (for example, by reducing the number of objects fetched per request or switching to using a watch request) can help reduce the total concurrency required by the given workload.
```

Comment:
Something that may be worth to add:
Isolating components by creating new flowschema (and PLC) since a component starves other components in the same flow.
You can polish the wording.